jquery.flot.logaxis.js

```diff
@@ -98,6 +87,10 @@ Set axis.mode to "log" to enable.
             }
         });
 
+        if (maxIdx === -1) {
+            maxIdx = PREFERRED_LOG_TICK_VALUES.length - 1;
+        }
+
```

Comment:
For numbers greater than Number.MAX_VALUE, make maxIdx the one corresponding to MAX_VALUE